lib/paperclip/thumbnail.rb

```diff
@@ -3,7 +3,7 @@ module Paperclip
   class Thumbnail < Processor
 
     attr_accessor :current_geometry, :target_geometry, :format, :whiny, :convert_options,
-                  :source_file_options, :animated, :auto_orient
+                  :source_file_options, :animated, :auto_orient, :pre_convert_options
```

Comment:
Is someone expecting me to address this before considering the pull request for merging, because there are several other lines in this same file that do not meet this style requirement?